test-requirements.txt

```diff
@@ -1,2 +1,10 @@
 dbus-python
-jinja2
```

Comment:
Was this used during the config conversion? And we're no longer doing that during tests? Do we need a test for that specifically?